File: ts/iq/encrypt~set~list+pq_list.ts
/** <iq type="set" xmlns="encrypt">. Replies with one of: Success, RequestError, ServerError. */
export function buildPreKeysDelete(): BinaryNode {
  return { tag: 'iq', attrs: { type: 'set', xmlns: 'encrypt', to: 's.whatsapp.net' }, content: [
      { tag: 'list', attrs: {  } }, /* repeated */
      { tag: 'pq_list', attrs: {  } }, /* repeated */
      { tag: 'op', attrs: { mode: 'delete' } },
    ] }
}

File: go/iq/encrypt~set~list+pq_list.go
// BuildPreKeysDelete builds <iq type="set" xmlns="encrypt">. Replies with one of: Success, RequestError, ServerError.
func BuildPreKeysDelete() waBinary.Node {
	return waBinary.Node{
		Tag: "iq",
		Attrs: waBinary.Attrs{"type": "set", "xmlns": "encrypt", "to": "s.whatsapp.net"},
		Content: []waBinary.Node{
			{Tag: "list", Attrs: waBinary.Attrs{}}, // repeated
			{Tag: "pq_list", Attrs: waBinary.Attrs{}}, // repeated
			{Tag: "op", Attrs: waBinary.Attrs{"mode": "delete"}},
		},
	}
}